README.md

```diff
@@ -1,6 +1,53 @@
-# FINDOLOGIC export toolkit
+FINDOLOGIC export toolkit
+=========================
 
 [![Travis](https://img.shields.io/travis/findologic/libflexport.svg)](https://travis-ci.org/findologic/libflexport)
 [![Latest Stable Version](https://poser.pugx.org/findologic/libflexport/v/stable)](https://packagist.org/packages/findologic/libflexport)
 
-**Highly experimental! Don't use it!**
\ No newline at end of file
+Synopsis
+========
+
+This project provides a export library for XML and CSV generation according to the FINDOLOGIC export patterns.
+* XML: <https://docs.findologic.com/doku.php?id=export_patterns:xml>
+* CSV: <https://docs.findologic.com/doku.php?id=export_patterns:csv>
```

Comment:
Stability of the export formats should be mentioned:

* XML: ☑️ 
* CSV: 😟 

Of course, phrasing should not be limited to an emoji.

Reply:
Done. Mentioned a few XML advantages over CSV. Maybe you could add some more?

Reply:
The critical advantage is that it supports any text encoding, validation is more reliable, escaping is simpler, and the structure is standardized instead of proprietary.

Reply:
Done